JavaScript/wh7_tech skills/hw_m2/javascript.js

```diff
@@ -0,0 +1,22 @@
+// 1
+let num = prompt('Укажите число', '');
+let exp = prompt('Укажите степень', '');
+
+console.log(`${num} в степени ${exp} = ${Math.pow(num, exp)}`);
```

Comment:
in this task you should create function pow with same functionality as Math.pow, but without Math.pow